fluentd/configs.d/input-ocp-audit.conf

```diff
@@ -0,0 +1,29 @@
+<source>
+  @type tail
+  path /var/log/audit/audit-ocp.log
```

Comment:
This path is no longer available in OCP4.X.
However, there are audit logs available in the following locations:
* /var/log/kube-apiserver/audit.log
* /var/log/openshift-apiserver/audit.log